src/sql/src/plan/statement/ddl.rs

```diff
@@ -767,12 +767,14 @@ pub fn plan_create_source(
             subscribe_key,
             channel,
         } => {
-            scx.require_experimental_mode("PubNub Sources")?;
```

Comment:
why did this go away?

Reply:
Hah, that's the point of this whole ordeal! Now that PubNub sources are marked as volatile, folks are comfortable with them not being experimental.